httpbp/middlewares.go

```diff
@@ -287,6 +287,9 @@ func SupportedMethods(method string, additional ...string) Middleware {
 // bubble up into other middlewares. Since it is always added to the middleware
 // chain is a specific position, it is not exported.
 func recoverPanic(name string, next HandlerFunc) HandlerFunc {
+	promCounter := panicRecoverCounter.With(prometheus.Labels{
```

Comment:
🔕 since 'prom' is the way of the future, can we just call it "counter" or "metric"?